### PR TITLE
Add checkmarks on sequence page [UI Change]

### PR DESCRIPTION
**Pull request by Raemon**
This is a revamp of a previous PR. It does two things: make SequencesPages match the look of the LargeSequencesItem, and as part of that process also unifies how Chapter Titles works.

It also adds a comment clarifying that SectionTitle is really meant to be used in one particular place and shouldn't be used as a generic title component in other contexts.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203068488025640) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/sequences/ChapterTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    ...theme.typography.chapterTitle,
+    margin: "1em 0",
+  }
+});
+
+const ChapterTitle = ({classes, title}: {
+  classes: ClassesType,
+  title?: string
+}) => {
+  if (!title) return null
```
Comment (b0b3rt) [inline packages/lesswrong/components/sequences/ChapterTitle.tsx:19]: I think I'd rather externalize this and require the caller to check if they have a title before trying to render the component

**Comment by Raemon** [comment]
Here's what it looks like for EA Forum:

image

File: packages/lesswrong/components/sequences/SequencesHoverOver.tsx [hunk 2]
```diff
@@ -74,13 +74,16 @@ export const SequencesHoverOver = ({classes, sequence, showAuthor=true}: {
...
     {/* show a loading spinner if either sequences hasn't loaded or chapters haven't loaded */}
     {(!sequence || (!chapters && chaptersLoading)) && <Loading/>}
-    {sequence && posts.map(post => 
-      <SequencesSmallPostLink 
-        key={sequence._id + post._id} 
-        post={post}
-        sequenceId={sequence._id}
-      />
-    )}
+    {sequence && chapters?.flatMap(chapter => {
```
Comment (b0b3rt) [inline packages/lesswrong/components/sequences/SequencesHoverOver.tsx:77]: `chapters?.map` should be sufficient, I think?  We aren't returning an array that we want to flatten from inside it

**Review by b0b3rt — APPROVED** [review]
apart from a minor `flatMap` > `map` question, lgtm - I'd probably want to get someone from the EA forum to check out their styling, though

**Pull request merged**